### PR TITLE
Add raycast-notification extension

### PR DESCRIPTION
**Pull request by maxnyby**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by maxnyby** [comment]
I noticed that you write on https://manual.raycast.com/extensions:
> Your extension does more than use any of Raycast's native functionalities like Quicklinks, Snippets, Clipboard History, Calculator, etc;

So I completely understand if you reject this extension. Id run it as just a local extension, but
1. Deeplinks don't seem to work with local extensions, and
2. I've seen other online on the quest for this functionality, because your HUD design is so much nicer than the default "osascript display notification"-one, but the functionality doesn't seem to be built in to raycast at this moment.

So I hope you'll approve this extension, or at least point me in the right direction to get deeplinks working on a local extension.

File: extensions/raycast-notification/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "raycast-notification",
+  "title": "Raycast Notification",
+  "description": "This extension makes it easy to display raycast notifications via a quicklink, making the nice raycast interface usable to other applications or scripts",
```
Comment (pernielsentikaer) [inline extensions/raycast-notification/package.json:5]:
```suggestion
  "description": "This extension makes it easy to display Raycast notifications via a quicklink, making the Raycast interface accessible to other applications or scripts",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Your extension is missing an icon. You can create one using our generator here: https://ray.so/icon

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**